LIMS2DB/classes.py

```diff
@@ -418,159 +353,81 @@ def build(self):
-                            "sequencing"
-                        ][seq.luid] = {}
-                        self.obj["projects"][project_luid]["samples"][sample_name][
-                            "sequencing"
-                        ][seq.luid]["date"] = seq.daterun.strftime("%Y-%m-%d")
-
-                        query = "select art.* from artifact art \
+                        self.obj["projects"][project_luid]["samples"][sample_name]["sequencing"][seq.luid] = {}
+                        self.obj["projects"][project_luid]["samples"][sample_name]["sequencing"][seq.luid]["date"] = seq.daterun.strftime("%Y-%m-%d")
+
+                        query = f"select art.* from artifact art \
                                 inner join processiotracker piot on piot.inputartifactid=art.artifactid \
                                 inner join artifact_ancestor_map aam on aam.artifactid=art.artifactid \
-                                where piot.processid={processid} and aam.ancestorartifactid={ancestorid};".format(
-                            processid=seq.processid, ancestorid=out.artifactid
-                        )
+                                where piot.processid={seq.processid} and aam.ancestorartifactid={out.artifactid};"
 
-                        seq_inputs = (
-                            self.session.query(Artifact)
-                            .from_statement(text(query))
-                            .all()
-                        )
+                        seq_inputs = self.session.query(Artifact).from_statement(text(query)).all()
                         seq_qc_flag = "UNKNOWN"
                         for seq_inp in seq_inputs:
-                            if (
-                                seq_qc_flag != "FAILED"
-                            ):  # failed stops sequencing update
+                            if seq_qc_flag != "FAILED":  # failed stops sequencing update
                                 seq_qc_flag = seq_inp.qc_flag
 
-                        self.obj["projects"][project_luid]["samples"][sample_name][
-                            "sequencing"
-                        ][seq.luid]["status"] = seq_qc_flag
+                        self.obj["projects"][project_luid]["samples"][sample_name]["sequencing"][seq.luid]["status"] = seq_qc_flag
                         # updates every time until the latest one, because of the order by in fetching sequencing processes.
-                        self.obj["projects"][project_luid]["samples"][sample_name][
-                            "sequencing_status"
-                        ] = seq_qc_flag
+                        self.obj["projects"][project_luid]["samples"][sample_name]["sequencing_status"] = seq_qc_flag
 
 
 class ProjectSQL:
-    def __init__(
-        self, session, log, pid, host="genologics.scilifelab.se", couch=None, oconf=None
-    ):
+    def __init__(self, session, log, pid, host="genologics.scilifelab.se", couch=None, oconf=None):
```

Comment:
We should probably change the default host, maybe in another PR

Reply:
That works too

Reply:
[97069c9](https://github.com/NationalGenomicsInfrastructure/LIMS2DB/pull/171/commits/97069c9c4b98b8a476e7a55257571939b6a5f6c0)